### ISSUE TITLE
[Nonprofit Site Guide] Tweak accessibility report-related content

### ISSUE THREAD

**Issue by clatwell** [Good-Bids/goodbids#697]
As it turns out, the Accessibility Checker Pro plugin can only scan pages if they're accessible to the public. Currently, Nonprofit Site front-pages are not accessible to site visitors unless the site is live. So, we'd like to adjust some of our Setup Guide content to better inform Admins about this wrinkle. 

### Confirm Site Accessibility content tweak 

Update description for Design and Content > Confirm Site Accessibility action item to: All GOODBIDS Nonprofit sites are required to meet accessibility guidelines. Once your site is live, the Accessibility Checker Pro plugin will be able to scan your pages for issues. After launch, review the site-wide accessibility report and resolve any issues flagged on your content and auction pages. 

### Update Site Status content 

**Pending** 

- Remove note about site-wide accessibility report; first sentence should read: Before you launch your site, be sure to confirm your site timezone and update the placeholder content on your homepage. 

**Live** 

- Add a new sentence at the top: "Now that your site is live, review the [site-wide accessibility report](/) and resolve any issues flagged on your pages." (Link goes to /wp-admin/admin.php?page=accessibility_checker)
